src/version.h

```diff
@@ -4,5 +4,5 @@
 */
 
 #ifndef VERSION_STRING
-#define VERSION_STRING "@PACKAGE_GIT_VERSION@";
+#define VERSION_STRING "201807_Kaikoura-355-g0c46873 (modified)";
```

Comment:
Please revert this change.

Reply:
This was unintentional. Thanks for pointing it out.